app/models/user.rb

```diff
@@ -0,0 +1,23 @@
+class User < ApplicationRecord
+  rolify
+  before_create :set_role
+  mount_uploader :avatar, AvatarUploader
+  has_many :orders
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  validates_presence_of :email
+  validates_presence_of :avatar
+  validates_integrity_of :avatar
+  validates_processing_of :avatar
+
+  private
+
+  def set_role
+    if id == 1
```

Comment:
if you remove all users and then create next user - he'll have id: 2

`add_role(:admin) if self.class.exists?` (Good)
Move to `form object` (Best)